docker-compose.yml

```diff
@@ -33,3 +35,23 @@ services:
     image: "redis"
     ports:
       - "6379:6379"
+  elasticsearch:
+    image: docker.elastic.co/elasticsearch/elasticsearch:7.5.2
+    container_name: elasticsearch
+    environment:
```

Comment:
These settings came from quite a bit of trial and error and reading (often outdated) blog posts, so let's not change them too much as long as they work.

Reply:
Don't try to fix something that ain't broke 😉